.buildkite/pipeline.yml

```diff
@@ -430,40 +427,41 @@ steps:
         format: "junit"
         branch: "^master|next$$"
     concurrency: 25
-    concurrency_group: 'browserstack-app'
+    concurrency_group: 'bitbar'
     concurrency_method: eager
     retry:
       automatic:
         - exit_status: -1  # Agent was lost
           limit: 2
 
   # PLAT-11155: App hang tests run on BrowserStack (Appium 1.x) for now
-  - label: ':browserstack: iOS 17 app hang tests'
+  - label: ':bitbar: iOS 17 app hang tests'
     depends_on:
       - cocoa_fixture
     timeout_in_minutes: 30
     agents:
       queue: opensource
     plugins:
       artifacts#v1.5.0:
-        download: "features/fixtures/ios/output/ipa_url_bs_release.txt"
+        download: "features/fixtures/ios/output/ipa_url_bb_release.txt"
         upload: "maze_output/failed/**/*"
       docker-compose#v3.7.0:
-        pull: cocoa-maze-runner
-        run: cocoa-maze-runner
+        pull: cocoa-maze-runner-bitbar
+        run: cocoa-maze-runner-bitbar
         command:
-          - "--app=@build/ipa_url_bs_release.txt"
-          - "--farm=bs"
+          - "--app=@build/ipa_url_bb_release.txt"
+          - "--farm=bb"
           - "--device=IOS_17"
-          - "--appium-version=1.21.0"
+          - "--no-tunnel"
+          - "--aws-public-ip"
           - "--fail-fast"
           - "features/app_hangs.feature"
       test-collector#v1.10.2:
         files: "reports/TEST-*.xml"
         format: "junit"
         branch: "^master|next$$"
     concurrency: 5
-    concurrency_group: 'browserstack-app'
+    concurrency_group: 'bitbar'
```

Comment:
The App Hang tests will stay on BrowserStack due to the nature of those tests and issues with Appium 2.